test/test_utilities.cpp

```diff
@@ -66,6 +68,32 @@ void AssignInputValues(Tensor& A, float x) {
   }
 }
 
+// Input x will be used as an anchor
+// Actual value assigned equals to x * i
+void AssignInputValuesAnchor(Tensor& A, float x) {
+  auto A_flat = A.flat<float>();
+  auto A_flat_data = A_flat.data();
+  for (int i = 0; i < A_flat.size(); i++) {
+    A_flat_data[i] = x * i;
+  }
+}
+
+// Randomly generate a float number between -0.0005 ~ 0.0005
+void AssignInputValuesRandom(Tensor& A) {
```

Comment:
The comment says this function generates a float between -0.0005 ~0.0005. Following the other comments, the range seems different.

Reply:
Also it would be helpful to have a more generic function that could generate numbers between any range

Reply:
Yes, I found out this discrepancy in comments yesterday, accidentally pushed to another branch. Will do a pull request with that branch today.
Sure, I am thinking about that too. I will do another pull request regarding adding another function to random generate numbers in range. Thanks @shresthamalik .